pom.xml

```
@@ -43,12 +43,12 @@
             <dependency>
                 <groupId>uk.ac.ebi.ampt2d</groupId>
                 <artifactId>accession-commons-mongodb</artifactId>
-                <version>0.7.0</version>
+                <version>0.7.1-SNAPSHOT</version>
```

Don't depend on snapshot versions on libraries. This is a recipe for disaster.
```suggestion
                <version>0.7.1</version>
```

---

Agree, I have it as a comment in the PR description to not merge until this is changed